src/test/javascript/spec/component/assessment-shared/assessment-locks.component.spec.ts

```diff
@@ -31,24 +55,64 @@ describe('AssessmentLocksComponent', () => {
                 { provide: LocalStorageService, useClass: MockSyncStorage },
                 { provide: SessionStorageService, useClass: MockSyncStorage },
                 { provide: TranslateService, useClass: MockTranslateService },
+                { provide: textAssessmentService, useClass: TextAssessmentService },
+                { provide: programmingAssessmentService, useClass: ProgrammingAssessmentManualResultService },
+                { provide: fileUploadAssessmentService, useClass: FileUploadAssessmentService },
+                { provide: modelingAssessmentService, useClass: ModelingAssessmentService },
             ],
-        }).compileComponents();
+        })
+            .compileComponents()
+            .then(() => {
+                programmingAssessmentService = TestBed.inject(ProgrammingAssessmentManualResultService);
+                textAssessmentService = TestBed.inject(TextAssessmentService);
+                fileUploadAssessmentService = TestBed.inject(FileUploadAssessmentService);
+                modelingAssessmentService = TestBed.inject(ModelingAssessmentService);
+                courseService = TestBed.inject(CourseManagementService);
+            });
     });
 
     beforeEach(() => {
         fixture = TestBed.createComponent(AssessmentLocksComponent);
         component = fixture.componentInstance;
         fixture.detectChanges();
+        jest.resetAllMocks();
+        windowConfirmStub = jest.spyOn(window, 'confirm').mockReturnValue(true);
     });
-
     it('should create', () => {
```

Comment:
This test case can be omitted. Afaik we agreed on that this test case is trivial and we try to avoid `toBeTruthy()` in general

Reply:
okay :). i removed this old test